index.js

```diff
@@ -1,16 +1,43 @@
+// importeer een tweetal functies en deze worden opgeslagen in de variabelen die tussen de {} staan
+
 const { question } = require("readline-sync");
-const { displayWordSoFar, isGameWon, isGameLost } = require("./gamelogic");
+const {
+  counter,
+  displayWordSoFar,
+  isGameWon,
+  isGameLost,
+} = require("./gamelogic");
 
+// de functie wordt hier gedeclareerd maar nog niet uitgevoerd
 function game(word, guesses) {
   console.log("Dit heb je tot nu toe geraden: ", guesses);
+  const letter = question("Raad een letter: ").toLowerCase();
 
-  const letter = question("Raad een letter: ");
+  // Geeft een melding aan de gebruiker wanneer hij meer dan 1 letterinvoertg
+  if (letter.length > 1) {
+    console.log("Je mag maar 1 letter invoeren, probeer het opnieuw");
+    return game(word, guesses);
+  }
 
   // voeg de geraden letter toe aan de array met guesses
   guesses.push(letter);
 
-  // volgende ronde! we roepen game nog een keer aan
-  game(word, guesses);
+  // laat het huidige woord en het galgje zien
+
+  console.log(`\n\n Het galgje \n\n ${counter(word, guesses)}\n`);
```

Comment:
Dit is de enige plek waar je 'back-ticks'  gebruikt. Waarom hier wel?
Bij Nova gebruiken we deze nog niet...

Reply:
Het verschil tussen de lessen van Nova en Mentor is volgens mij best groot. ik heb de afgelopen 3x les gehad van Mentor en hij geeft ontzettend veel info in zijn lessen, veel wat ook niet voorkomt in edhub. 

de reden weet ik niet eigenlijk ;) niet echt consistent he. Ben wel van plan om het standaard te gaan gebruiken, vindt het namelijk wel overzichtelijk.